examples/basic/annotations/slope.py

```diff
@@ -16,7 +16,10 @@
 p.circle(xpts, ypts, size=6, alpha=0.6, fill_color=None)
 
 slope = Slope(gradient=slope, y_intercept=intercept,
-              line_color='orange', line_dash='dashed', line_width=4)
+              line_color='orange', line_dash='dashed', line_width=4,
+              above_fill_color='orange',
+              below_fill_color='red',
+              )
```

Comment:
There is not a single incidence of this formatting anywhere to my knowledge. For examples especially this is unacceptable, as vertical space is typically what is at a premium in displayed docs. 
```suggestion
              line_color='orange', line_dash='dashed', line_width=4,
              above_fill_color='orange', below_fill_color='red')
```

Reply:
Actually, please revert these changes entirely. Keeping them would require updating the gallery thumbnail. I prefer the clean look of the current example. If you want to have a specific example for the user guide then adding a new slope example for that purpose would be acceptable.

Reply:
That was committed accidentally.